anesthetic/plot.py

```diff
@@ -59,314 +145,428 @@ def _constructor(self):
     def _constructor_sliced(self):
         return AxesSeries
 
-    def axlines(self, params, values, **kwargs):
-        """Add vertical and horizontal lines across all axes.
+    @staticmethod
+    def _position_frame(index, columns, lower, diagonal, upper):
+        """Compute positions with lower=-1, diagonal=0, upper=+1."""
+        data = np.full((np.size(index), np.size(columns)), None)
+        position = pandas.DataFrame(data=data, index=index, columns=columns)
+        all_params = list(columns) + list(index)
+        for j, y in enumerate(index):
+            for i, x in enumerate(columns):
+                if all_params.index(x) < all_params.index(y):
+                    if lower:
+                        position[x][y] = -1
+                elif all_params.index(x) > all_params.index(y):
+                    if upper:
+                        position[x][y] = +1
+                elif diagonal:
+                    position[x][y] = 0
+        return position
```

Comment:
Is it OK if I have a look at turning this into a function, rather than an object, and push to this branch?

Reply:
Agreed.